src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -312,7 +312,8 @@ public static bool IsLinuxConsumptionOnAtlas(this IEnvironment environment)
         public static bool IsLinuxConsumptionOnLegion(this IEnvironment environment)
         {
             return !environment.IsAppService() &&
-                   !string.IsNullOrEmpty(environment.GetEnvironmentVariable(ContainerName)) &&
```

Comment:
There are existing tests for this helper in EnvironmentTests that should be updated to test this new behavior

Reply:
Updated `EnvironmentExtensionsTests`